src/components/sequencing/CommandPanel/CommandDictionary.svelte

```diff
@@ -0,0 +1,283 @@
+            command.arguments.find(argument => {
+              return argument.name.toLowerCase().includes(searchValue);
+            }) !== undefined
+          );
+        });
+        break;
+      }
+      case 'command_name':
+      default: {
+        commandResults = commandList.filter(command => {
+          return command.stem.toLowerCase().includes(searchValue);
+        });
+        break;
+      }
+    }
+  }
+
+  function onClear() {
+    dispatch('selectCommandDefinition', null);
+  }
+
+  async function onCopy() {
+    if (selectedCommandDefinition) {
+      try {
+        await navigator.clipboard.writeText(`${selectedCommandDefinition.stem}`);
+        showSuccessToast(`"${selectedCommandDefinition.stem}" copied to clipboard`);
+      } catch {
+        showFailureToast(`Error copying "${selectedCommandDefinition.stem}" to clipboard`);
+      }
+    }
+  }
+
+  function onSearch(event: Event) {
+    const { value } = getTarget(event);
+    searchValue = `${value}`.toLowerCase();
+  }
+
+  function onSelectCommandDefinition(commandDefinition: FswCommand | HwCommand) {
+    dispatch('selectCommandDefinition', commandDefinition);
+  }
+</script>
+
+<div class="command-dictionary-container">
+  <div class="dictionary-name">
+    <div class="breadcrumbs-container">
+      {#if selectedCommandDefinition !== null}
+        <button class="st-button-link breadcrumb" on:click={onClear}>{commandDictionary.id}</button>
+        <ChevronRightIcon />
+        <div class="breadcrumb">{selectedCommandDefinition.stem}</div>
+      {:else}
+        {commandDictionary.id}
+      {/if}
+    </div>
+    <div class="breadcrumbs-icons">
+      {#if selectedCommandDefinition !== null}
+        <button
+          class="st-button-link"
+          on:click={onCopy}
+          use:tooltip={{ content: 'Copy command stem to clipboard', placement: 'top' }}
+        >
+          <CopyIcon />
+        </button>
+        <button
+          class="st-button-link"
+          on:click={onClear}
+          use:tooltip={{ content: 'Deselect command', placement: 'top' }}
+        >
+          <CloseIcon />
+        </button>
+      {/if}
+    </div>
+  </div>
+  <div class="dictionary-search" class:hidden={selectedCommandDefinition !== null}>
+    <label for="dictionary-search">Command Dictionary</label>
+    <div class="search-input">
+      <Input>
+        <div class="search-icon" slot="left"><SearchIcon /></div>
+        <input
+          class="st-input w-100"
+          name="dictionary-search"
+          placeholder={`Search ${searchFieldMap[searchField]}`}
+          on:input={onSearch}
+        />
+      </Input>
+      <select bind:value={searchField} class="st-select w-100">
+        <option value={'command_name'}>Command name</option>
+        <option value={'enum'}>Enum</option>
+        <option value={'argument_name'}>Argument name</option>
+      </select>
+    </div>
+    <div class="dictionary-results">
+      {#each commandResults as commandResult}
```

Comment:
Could you add empty states for these result sections for when no items match your filter?